tests/unit/test_clidriver.py

```diff
@@ -620,17 +621,23 @@ def test_aws_with_endpoint_url(self):
         self.assert_params_for_cmd(
             'ec2 describe-instances --endpoint-url https://foobar.com/',
             expected_rc=0)
-        self.create_endpoint.assert_called_with(
-            mock.ANY, 'us-east-1', verify=None, endpoint_url='https://foobar.com/',
-            is_secure=True, response_parser_factory=mock.ANY)
+        call_args = self.create_endpoint.call_args
+        self.assertEqual(call_args[0], (mock.ANY, 'us-east-1'))
+        expected = dict(
+            timeout=(DEFAULT_TIMEOUT, DEFAULT_TIMEOUT),
+            verify=None, endpoint_url='https://foobar.com/', is_secure=True)
+        self.assertTrue(set(expected.items()) <= set(call_args[1].items()))
```

Comment:
Rather than do this, why not take the same approach you did with just verifying the specific kwarg the test is interested in?  So here we'd just check kwargs['endpoint_url']?